.k8s_ci.Dockerfile

```diff
@@ -115,7 +153,8 @@ RUN set -xe; \
 WORKDIR $ROOT_DIR
 # Copy galaxy files to final image
 # The chown value MUST be hardcoded (see #35018 at github.com/moby/moby)
-COPY --chown=galaxy:galaxy --from=stage1 $ROOT_DIR .
+COPY --chown=galaxy:galaxy --from=server_build $ROOT_DIR .
+COPY --chown=galaxy:galaxy --from=client_build $SERVER_DIR/static ./server/static
```

Comment:
@dannon @mvdbeek This copies only the static files over from the client build. Anything else we should copy over?
I guess we do need node for the expression tools (see comment above)

Reply:
Check visualization build artifacts (they get staged into `/static/plugins/` during the build, but without testing it this should probably work.

Reply:
Thanks, `/static/plugins` dir seems to be populated ok.